get_deps.sh

```diff
@@ -74,100 +77,108 @@ fi
 			TF_ARCH=x86_64
 			LIBTF_URL_BASE=https://storage.googleapis.com/tensorflow/libtensorflow
-		elif [[ $ARCH == arm64v8 ]]; then
-			TF_VERSION=1.14.0
-			TF_ARCH=arm64
-			LIBTF_URL_BASE=https://s3.amazonaws.com/redismodules/tensorflow
-		elif [[ $ARCH == arm32v7 ]]; then
-			TF_VERSION=1.14.0
-			TF_ARCH=arm
-			LIBTF_URL_BASE=https://s3.amazonaws.com/redismodules/tensorflow
 		fi
-	elif [[ $OS == macosx ]]; then
-		TF_VERSION=1.14.0
-		TF_OS=darwin
-		TF_BUILD=cpu
-		TF_ARCH=x86_64
-		LIBTF_URL_BASE=https://storage.googleapis.com/tensorflow/libtensorflow
-	fi
 
-	LIBTF_ARCHIVE=libtensorflow-${TF_BUILD}-${TF_OS}-${TF_ARCH}-${TF_VERSION}.tar.gz
+		LIBTF_ARCHIVE=libtensorflow-${TF_BUILD}-${TF_OS}-${TF_ARCH}-${TF_VERSION}.tar.gz
 
-	[[ ! -f $LIBTF_ARCHIVE || $FORCE == 1 ]] && wget --quiet $LIBTF_URL_BASE/$LIBTF_ARCHIVE
+		[[ ! -f $LIBTF_ARCHIVE || $FORCE == 1 ]] && wget --quiet $LIBTF_URL_BASE/$LIBTF_ARCHIVE
 
-	rm -rf $LIBTENSORFLOW.x
-	mkdir $LIBTENSORFLOW.x
-	tar xf $LIBTF_ARCHIVE --no-same-owner --strip-components=1 -C $LIBTENSORFLOW.x
-	mv $LIBTENSORFLOW.x $LIBTENSORFLOW
-	
-	echo "Done."
+		rm -rf $LIBTENSORFLOW.x
+		mkdir $LIBTENSORFLOW.x
+		tar xf $LIBTF_ARCHIVE --no-same-owner -C $LIBTENSORFLOW.x
+		mv $LIBTENSORFLOW.x $LIBTENSORFLOW
+		
+		echo "Done."
+	else
+		echo "TensorFlow is in place."
+	fi
 else
-	echo "TensorFlow is in place."
-fi
+	echo "Skipping TensorFlow."
+fi # WITH_TF
 
 ###################################################################################### LIBTORCH
 
 PT_VERSION="1.2.0"
 
-[[ $FORCE == 1 ]] && rm -rf $LIBTORCH
-
-if [[ ! -d $LIBTORCH ]]; then
-	echo "Installing libtorch ..."
-
-	if [[ $OS == linux ]]; then
-		PT_OS=linux
-		if [[ $GPU == no ]]; then
-			PT_BUILD=cpu
-		else
-			PT_BUILD=cu100
-		fi
-		if [[ $ARCH == x64 ]]; then
+if [[ $WITH_PT != 0 ]]; then
+	[[ $FORCE == 1 ]] && rm -rf $LIBTORCH
+
+	if [[ ! -d $LIBTORCH ]]; then
+		echo "Installing libtorch ..."
+
+		if [[ $OS == linux ]]; then
+			PT_OS=linux
+			if [[ $GPU == no ]]; then
+				PT_BUILD=cpu
+			else
+				PT_BUILD=cu100
+			fi
+			if [[ $ARCH == x64 ]]; then
+				PT_ARCH=x86_64
+			elif [[ $ARCH == arm64v8 ]]; then
+				PT_ARCH=arm64
+			elif [[ $ARCH == arm32v7 ]]; then
+				PT_ARCH=arm
+			fi
+		elif [[ $OS == macosx ]]; then
+			PT_OS=macos
 			PT_ARCH=x86_64
-		elif [[ $ARCH == arm64v8 ]]; then
-			PT_ARCH=arm64
-		elif [[ $ARCH == arm32v7 ]]; then
-			PT_ARCH=arm
+			PT_BUILD=cpu
 		fi
-	elif [[ $OS == macosx ]]; then
-		PT_OS=macos
-		PT_ARCH=x86_64
-		PT_BUILD=cpu
-	fi
 
-	[[ "$PT_VERSION" == "latest" ]] && PT_BUILD=nightly/${PT_BUILD}
+		[[ "$PT_VERSION" == "latest" ]] && PT_BUILD=nightly/${PT_BUILD}
 
-	LIBTORCH_ARCHIVE=libtorch-${PT_BUILD}-${PT_OS}-${PT_ARCH}-${PT_VERSION}.tar.gz
-	LIBTORCH_URL=https://s3.amazonaws.com/redismodules/pytorch/$LIBTORCH_ARCHIVE
+		LIBTORCH_ARCHIVE=libtorch-${PT_BUILD}-${PT_OS}-${PT_ARCH}-${PT_VERSION}.tar.gz
+		LIBTORCH_URL=https://s3.amazonaws.com/redismodules/pytorch/$LIBTORCH_ARCHIVE
```

Comment:
It was the same way earlier, but I notice there's no codepath that downloads LIBTORCH from the official PyTorch packages for x86, like
https://download.pytorch.org/libtorch/cpu/libtorch-macos-1.2.0.zip
or
https://download.pytorch.org/libtorch/cu100/libtorch-shared-with-deps-1.2.0.zip
https://download.pytorch.org/libtorch/cpu/libtorch-shared-with-deps-1.2.0.zip

I think we should do it as we do for TensorFlow and ONNXRuntime

Reply:
Fixed.